src/components/GeneOntologyNetwork.js

```diff
@@ -60,8 +95,55 @@ function GeneOntologyNetwork({ data }) {
 				idealEdgeLength: 100
 			}
 		});
+		let div;
+		let node = cy.elements().nodes();
+		node.unbind('mouseover');
+		node.bind('mouseover', event => {
+			const {
+				data: { info }
+			} = event.target[0]._private;
+			let content;
+			document.body.style.cursor = 'pointer';
+			if (info.class === 'Gene') {
+				event.target.style('backgroundColor', '#666666');
+				content = `
+					<div>
+						<span>Symbol: </span><strong>${info.symbol}</strong><br/><div style="padding: 2px"></div>
+						<span>Name: </span><strong>${info.shortName}</strong><br/><div style="padding: 2px"></div>
+						<span>Primary Identifier: </span><strong>${info.primaryIdentifier}</strong><br/><div style="padding: 2px"></div>
+						<span>Secondary Identifier: </span><strong>${info.secondaryIdentifier}</strong>
+					</div>
+				`;
+			}
+			if (info.class === 'GOAnnotation') {
+				event.target.style('backgroundColor', '#EDBE05');
+				content = `
+					<div>
```

Comment:
Could you shift the content generation in a separate function which just takes the data and conditionally returns your the required `content`. It'll make the code look a little cleaner.